app/src/main/java/org/fossasia/openevent/app/common/utils/core/ErrorUtils.java

```diff
@@ -0,0 +1,39 @@
+package org.fossasia.openevent.app.common.utils.core;
+
+import retrofit2.HttpException;
+
+public final class ErrorUtils {
+
+    public static final int BAD_REQUEST = 400;
+    public static final int UNAUTHORIZED = 401;
+    public static final int FORBIDDEN = 403;
+    public static final int NOT_FOUND = 404;
+    public static final int METHOD_NOT_ALLOWED = 405;
+    public static final int REQUEST_TIMEOUT = 408;
+
+    private ErrorUtils() {
+        // Never Called
+    }
+
+    public static String getMessage(Throwable throwable) {
+        if (throwable instanceof HttpException) {
+            switch (((HttpException) throwable).code()) {
+                case BAD_REQUEST:
+                    return "Something went wrong! Please check any empty field if a form.";
```

Comment:
Yes it can. Create another static method in UtilModel for that. context is static now